bitnami/influxdb/Chart.yaml

```diff
@@ -15,4 +15,4 @@ maintainers:
 name: influxdb
 sources:
   - https://github.com/bitnami/bitnami-docker-influxdb
-version: 0.4.8
+version: 0.4.9
```

Comment:
This should bump the minior version (`0.5.0`) but you're bumping the patch one

Reply:
I'll get versions someday...